### PR TITLE
enable repo-level project assignment

### PR DESCRIPTION
**Pull request by pygmypenguin**
Currently todo-to-issue only supports assigning issues to an organization-level project, or a user-level one. This should allow the action to assign issues to projects that are localized to a repo within an organization

### CODE REVIEW

**Comment by alstr** [comment]
Cool, thanks for the contribution! Just wondering have you tested this out and it works? And is it for the legacy style projects?

At some point in the future I would like to improve how projects work with the action in general, but I don't think the API (last time I checked) for the new style projects is quite there yet, and the `add-to-project` action doesn't seem to work properly (see #138).

If this works for the time being then I'm happy to add it.

**Comment by pygmypenguin** [comment]
> Just wondering have you tested this out and it works? And is it for the legacy style projects?

This is for classic projects, yes. my org is running github enterprise and we don't have the new-style project support yet, so i wrote this using the existing logic and known paths (e.g. i know that https://gitUrl/orgName/repoName/projects is a valid url). Unfortunately, because we lock down what actions can be used, I can't just test it within that context without several layers of approval (your app is approved, but mine being a fork of yours would need separate approval). for my personal github account, i can't create classic projects to test against because i don't already have a classic project (see [here](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/creating-a-project-board))

long story short, i haven't tested it unfortunately because right now i can't.

**Comment by pygmypenguin** [comment]
> and the `add-to-project` action doesn't seem to work properly (see #138).

according to their [documentation](https://github.com/actions/add-to-project), it appears actions/add-to-project will never work with classic projects. since enterprise users will likely need their own workaround for an issue, i think this would be a good stopgap

**Comment by alstr** [comment]
Fair enough, sorry I've not got around to it yet, I'll aim to shortly.

Also, in the README I noticed you'd added a bit about potentially using the `GITHUB_TOKEN`; I never had any luck with using the default one when I originally developed the functionality. Have you had this work at all (with user/org projects)?
